coderd/insights_test.go

```diff
@@ -922,6 +922,16 @@ func TestTemplateInsights_Golden(t *testing.T) {
 						}
 					},
 				},
+				{
+					name: "weekly aggregated deployment wide",
+					makeRequest: func(templates []*testTemplate) codersdk.TemplateInsightsRequest {
+						return codersdk.TemplateInsightsRequest{
+							StartTime: frozenWeekAgo.AddDate(0, 0, -3),
+							EndTime:   frozenWeekAgo.AddDate(0, 0, 4),
```

Comment:
frozenWeekAgo is defined as UTC, are we only accepting UTC input to this?

Reply:
I changed the location in the other place. Here we have also a test covering `frozenWeekAgoSaoPaulo`.